src/rgw/setup/rgw_service

```diff
@@ -37,9 +37,13 @@ while [ $# -gt 0 ];  do
     shift 1
 done
 
+log_dir="$(dirname "${log_file}")"
```

Comment:
this logic in shell script will have to be reworked in python after #54 merge

Reply:
Rebased the patch with latest main and updated the PR.